crates/node_binding/scripts/build.js

```diff
@@ -30,77 +26,46 @@ build().then((value) => {
 async function build() {
 	return new Promise((resolve, reject) => {
 		let args = [
-			"build",
 			"--platform",
 			"--dts",
 			"binding.d.ts",
 			"--no-js",
-			// "--no-const-enum",
 			"--no-dts-header",
-			"--pipe",
```

Comment:
it seems that `--pipe` removed and does not add back.
Maybe the reason why d.ts modified